onnx/defs/nn/utils.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "onnx/common/assertions.h"
+#include "onnx/defs/function.h"
+#include "onnx/defs/schema.h"
+
+namespace ONNX_NAMESPACE {
+namespace defs {
+namespace nn {
+namespace utils {
+
+/** Implements shape and type propagation for Attention (23-). */
+void AttentionPropagateElemTypeFromInputToOutput(InferenceContext& ctx);
```

Comment:
Is there any change or you just move it here?

Reply:
Just moved. It is common to old and new operator. I was hoping to do the same with the inline function but it looked like more complex to do.